src/enums.h

```diff
@@ -92,7 +92,8 @@ enum art_effect_passive : int {
     AEP_INT_UP, // Intelligence + 4
     AEP_ALL_UP, // All stats + 2
     AEP_SPEED_UP, // +20 speed
-    AEP_IODINE, // Reduces radiation
+    AEP_PBLUE, // Reduces radiation
+    AEP_IODINE, // Resist half radiation
```

Comment:
Adding artifact effects like that could break things (as in, saves).
Just renaming the AEP_IODINE to AEP_PBLUE would be safer. Especially considering you didn't implement it anywhere.